### PR TITLE
[4.0] com _CSP Options help link

### PR DESCRIPTION
**Pull request by brianteeman**
The string that will generate the correct link for the help button on the CSP Reports Options page was missing

This PR fixes that.

The actual help page doesnt exist yet but now the link exists

### CODE REVIEW

File: administrator/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -657,6 +657,7 @@ JHELP_COMPONENTS_COM_CACHE_OPTIONS="Components_Cache_Manager_Settings"
 JHELP_COMPONENTS_COM_CHECKIN_OPTIONS="Components_Check_in_Configuration"
 JHELP_COMPONENTS_COM_CONTACT_OPTIONS="Components_Contact_Manager_Options"
 JHELP_COMPONENTS_COM_CONTENT_OPTIONS="Components_Article_Manager_Options"
+JHELP_COMPONENTS_COM_CSP_OPTIONS="Components_CSP_Reports_Options"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.ini:660]: The following has `Csp` and with this PR `CSP`. They should be consistent.

`JHELP_COMPONENTS_CSP_REPORTS="Components_Csp_Reports"`

Comment (zero-24) [reply administrator/language/en-GB/en-GB.ini:660]:
```suggestion
JHELP_COMPONENTS_COM_CSP_OPTIONS="Components_Csp_Reports_Options"
```

**Review by zero-24 — CHANGES_REQUESTED** [review]
As suggested inline we should make it consistent. After that fine for me.

**Comment by brianteeman** [comment]
changed as requested to be consistent

**Comment by zero-24** [comment]
@brianteeman can you please double check? It looks like nothing new has been commited?

**Review by zero-24 — APPROVED** [review]

**Comment by zero-24** [comment]
Fine for me thanks @brianteeman

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c14f24f9e497435f991b517d48722fd12b6777dd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24347">issues.joomla.org/tracker/joomla-cms/24347</a>.</sub>

**Comment by zero-24** [comment]
Merged thanks.

**Comment by brianteeman** [comment]
thanks

**Pull request merged**